### ISSUE TITLE
`SIMD{n}: ElementaryFunctions` conformances

### ISSUE THREAD

**Issue by dan-zheng** [apple/swift-numerics#103]
`ElementaryFunctions` was added to apple/swift with [conformances for `SIMD` types](https://github.com/stephentyrone/swift/blob/8708c7b5819e45e1f44fafd4e0ad02affb3535ad/stdlib/public/core/MathFunctions.swift.gyb#L128-L172). It seems those conformances were dropped when `ElementaryFunctions` was moved to apple/swift-numerics.

Is it reasonable to re-add `SIMD{n}: ElementaryFunctions` conformances to apple/swift-numerics?

---

Context: https://github.com/tensorflow/swift/issues/404

**Comment by stephentyrone** [comment]
Yup, this is a very reasonable thing to add, I'll take a look shortly.
